boards/xtensa/esp32s3/esp32s3-devkit/src/Make.defs

```diff
@@ -49,13 +49,12 @@ ifeq ($(CONFIG_LCD_ST7735),y)
 CSRCS += esp32s3_st7735.c
 endif
 
-SCRIPTIN = $(SCRIPTDIR)$(DELIM)esp32s3.template.ld
 SCRIPTOUT = $(SCRIPTDIR)$(DELIM)esp32s3_out.ld
 
 .PHONY = context distclean
 
-$(SCRIPTOUT): $(SCRIPTIN) $(CONFIGFILE)
-	$(Q) $(CC) -isystem $(TOPDIR)/include -C -P -x c -E $(SCRIPTIN) -o $@
+$(SCRIPTOUT): $(LDSCRIPT_TEMPLATE) $(CONFIGFILE)
+	$(Q) $(CC) -isystem $(TOPDIR)/include -I $(BOARD_COMMON_DIR)$(DELIM)scripts -C -P -x c -E $(LDSCRIPT_TEMPLATE) -o $@
```

Comment:
don't need call $(CC) now, since arch/Makefile will do the same thing after https://github.com/apache/incubator-nuttx/pull/7208

Reply:
Thanks for the tip! I've modified the Makefiles, but now this PR depends on the fix from #7503 being merged first.